### PR TITLE
Module_1_hello_microverse_project

### PR DESCRIPTION
**Pull request by assadounto**
### 1. Project Requirements Summary

- Create a new GitHub repo.
- Create a new branch according to the [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) rules.
- Add a .gitignore file.
- Set up [a linter for HTML and CSSand make sure that any files that are not relevant are ignored by git.
- Add descriptive README file to your project - 
- Add 3 files (each one in a separate commit):
- A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
- A .css file that includes one class with styles for your header (make it in your favorite color).
- A test.md file with the text "This file should be ignored by git".
- Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
- Open a pull request.
 

### 2. Project includes linters and follows github flow. Added linters.yml file for light house test and demo link

Thank your for the Changes to the branch Projectcopy of Module_1_hello_microverse_project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,31 @@
...
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
```
Comment (phelian23) [inline README.md]: - [ ] A professional README should include instruction steps to run the app locally for the sake of the non-technical user. Kindly include steps to clone and run the app locally.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# .gitignore
+node_modules/
+.hintrc
+.stylelintrc.json
```
Comment (phelian23) [inline .gitignore]: - [ ] The `.hintrc` and `.stylelintrc.json` files should not be added to the .gitignore file kindly remove it

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @assadounto, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Good commit messages

## Required Changes ♻️

- [ ] Kindly add a summary to your pull request describing the introduced changes in more detail.

Capture

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by assadounto** [comment]
@phelian23 I have done most of the changes requested

**Comment by assadounto** [comment]
Hello,
Following the requested changes, I have  committed them. Please review

**Review by ShadyShawkat — APPROVED** [review]
## Hi @assadounto, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**